cmake/common_variables.cmake

```diff
@@ -1 +0,0 @@
-set(THRUST_FILECHECK_DATA_PATH "${THRUST_SOURCE}/internal/test")
```

Comment:
Please put it back here, see the comment on line 6 of CMakeLists.txt.

Reply:
Done.